modules/userId/index.js

```diff
@@ -576,6 +586,7 @@ export function init(config) {
 
   // exposing getUserIds function in global-name-space so that userIds stored in Prebid can be used by external codes.
   (getGlobal()).getUserIds = getUserIds;
+  (getGlobal()).getUserIdsAsEids = getUserIdsAsEids
```

Comment:
we should be consistent with the semicolon line endings, so please add one to line 589

Reply:
Added the missing semicolon :)